CefSharp.WinForms.Example/BrowserTabUserControl.cs

```diff
@@ -434,5 +434,42 @@ private void FindCloseButtonClick(object sender, EventArgs e)
         {
             ToggleBottomToolStrip();
         }
+
+        public void ShowDevToolsDocked()
+        {
+            if (browserSplitContainer.Panel2Collapsed)
+            {
+                browserSplitContainer.Panel2Collapsed = false;
+
+                if (devToolsPanel == null || devToolsPanel.IsDisposed)
+                {
+                    devToolsPanel = new Panel()
```

Comment:
I don't think the `Panel` should be initially created in the `Designer`, if we're creating a new instance of it then that doesn't really fit. Also makes the example harder to copy and paste.

Reply:
Sure, are you happy for me to create a field variable outside of the designer instead?

Reply:
I'd probably avoid it, just retrieve the instance from the Controls collection when required.